### PR TITLE
Adding rewrite method

### PR DESCRIPTION
**Pull request by malec-palec**
Added rewrite method allows setting URL redirection rules

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -208,6 +208,18 @@ Proxy.prototype = {
         this.doReqWithOptions(options, postData, cb);
     },
 
+    /*
+     *  matchRegex - A matching URL regular expression
+     *  replace - Replacement URL
+     */
+    rewrite: function(port, obj, cb) {
+        var data = '';
```
Comment (zzo) [inline index.js]: does this need to be:

var data = '?'
to start the query string?

Comment (malec-palec) [reply index.js]: I followed the example of `limit` method:
https://github.com/zzo/browsermob-node/blob/392e9cd81355d4d27d769c0230322ad38bfe0398/index.js#L154-L157

Works nicely for me, checked on a live project.

**Comment by zzo** [comment]
cool thanks!!

**Comment by malec-palec** [comment]
Is there something else (besides increasing patch version in package.json) needs to be done in order to get an updated version on npmjs.com?

**Pull request merged**